FAQ.md

```diff
@@ -29,6 +29,9 @@ A: Whilst we do not build Windows binaries directly, you should still be able to
 Q: I installed the app, but I can't log in - it asks me for password?
 A: Your password is derived from the `taker_seed` file, and the credentials can be seen it the taker logs right after startup.
 
+Q: Can I use use ItchySats on mobile?
+A: Whilst we currently don't provide iOS or Android apps, our UI is mobile-friendly. This means that you can connect from your mobile browser to the machine running ItchySats app. At the moment the easiest way to do it is to run Itchysats on [Umbrel](https://getumbrel.com). tip: If you use [Tailscale](https://tailscale.com/), you can trade on-the-go 😃 (without needing to be on the same local network).
```

Comment:
Umbrel/RaspiBlitz also exposes itchysats with an onion address

Reply:
I'll mention this, but tailscale approach is often recommended as superior way of interacting with umbrel apps on mobile.
tried both, can attest to that. :)